admin/actions.php

```diff
@@ -728,3 +744,37 @@ function hmbkp_ajax_cron_test() {
 
 }
 add_action( 'wp_ajax_hmbkp_cron_test', 'hmbkp_ajax_cron_test' );
+
+/**
+ * Spawn a request to `wp-cron.php` and return the response.
+ *
+ * This function is designed to mimic the functionality in `spawn_cron()` with the addition of returning
+ * the result of the `wp_remote_post()` request.
```

Comment:
Needs updating to reference `wp_remote_head` now that we've changed that in the function.